File: java/1013-o_maior.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class Main {
  public static void main(String[] args) throws IOException {
    int a, b, c, maior;

    Scanner scanner = new Scanner(System.in);
    a = scanner.nextInt();
    b = scanner.nextInt();
    c = scanner.nextInt();
    scanner.close();

    maior = (a+b+Math.abs(a-b))/2;
    maior = (maior + c + Math.abs(maior - c))/2;

    System.out.printf("%d eh o maior\n", maior);
  }
}

File: kotlin/1013-o_maior.kt
import java.util.*

fun main(args: Array<String>) {
  val VALORES = readLine()!!
  val (A,B,C) = VALORES.split(" ").map { it.toInt() }
  var MAIOR = ((A+B+Math.abs(A-B))/2)
  MAIOR = (MAIOR + C + Math.abs(MAIOR - C))/2
  println("$MAIOR eh o maior")
}
